app/src/main/java/com/mobileproto/hireddit/hireddit/visuals/SpeechFragment.java

```diff
@@ -44,15 +48,21 @@
     private ArrayList voiceInput;
     private Intent recognizerIntent;
     private SpeechRecognizer sr;
+    private boolean typeMode = false;
+    private boolean quietMode = false;
     private String link;
     private ViewGroup.LayoutParams cParams;
     private Integer radius;
 
+
     @Bind(R.id.listenButton) ImageView listenButton;
     @Bind(R.id.helloReddit) TextView helloReddit;
     @Bind(R.id.speechTextDisplay) TextView speechTextDisplay;
     @Bind(R.id.commentText) TextView commentText;
     @Bind(R.id.settingsButton) ImageView settingsButton;
+    @Bind(R.id.muteButton) ImageView muteButton;
+    @Bind(R.id.volumeOnButton) ImageView volumeOnButton;
```

Comment:
is it better to instead have drawables and replace the images on the button instead of switching the imageview?